### PR TITLE
LPD-1373 Use the ThemeDisplayLocale when creating the ddmFormRenderingContext

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPD-1373

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @vngoliveira 
Could you take a look at my changes, please? :smile:

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/ValidationDate.js [hunk 1]
```diff
@@ -41,7 +41,11 @@ const getDateOptionsByType = (label, name) => ({
 
 /* TODO: enforce parameter type consistency and remove this function */
 function getFromParameter(parameter, key, getLocalizedValue) {
-	let value = getLocalizedValue(parameter) ?? parameter;
+	let value = getLocalizedValue(parameter);
+
+	if (value === undefined) {
+		value = parameter[Liferay.ThemeDisplay.getLanguageId()];
+	}
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/ValidationDate.js:48]: @marcelabc you can change to:

```
let value = getLocalizedValue(parameter) ?? parameter[Liferay.ThemeDisplay.getLanguageId()];
```

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**